build.py

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env python3
-# Copyright 2018-2021 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# Copyright (c) 2020-2021, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
This needs to be 2020-21. Why are you added a comma?

Reply:
Actually the comma is required. Here are the regex that we check for (not including AFFILICATES change):
COPYRIGHT_YEAR_RE0 = 'Copyright \\(c\\) (20[0-9][0-9]), NVIDIA CORPORATION. All rights reserved.'
COPYRIGHT_YEAR_RE1 = 'Copyright \\(c\\) (20[0-9][0-9])-(20[0-9][0-9]), NVIDIA CORPORATION. All rights reserved.'

In main branch the comma is there so curious why it is not showing up here in this diff?